packages/odata-query-objects/src/operation/QFunction.ts

```diff
@@ -19,7 +20,7 @@ function compileUrlParams(params: FunctionParams | undefined, notEncoded: boolea
   const queryParams: string[] = [];
   const pathParams = Object.entries(params).map(([key, value]) => {
     [key, value] = notEncoded ? [key, value] : [encodeURIComponent(key), encodeURIComponent(value)];
```

Comment:
Sorry that I've missed that in my code review.

Use different variables here: `[safeKey, safeValue]`